src/vonage/proactive_connect.py

```diff
@@ -0,0 +1,183 @@
+        )
+
+    def clear_list(self, list_id: str):
+        return self._client.post(
+            self._client.proactive_connect_host(),
+            f'/v0.1/bulk/lists/{list_id}/clear',
+            params=None,
+            auth_type=self._auth_type,
+        )
+
+    def sync_list_from_datasource(self, list_id: str):
+        return self._client.post(
+            self._client.proactive_connect_host(),
+            f'/v0.1/bulk/lists/{list_id}/fetch',
+            params=None,
+            auth_type=self._auth_type,
+        )
+
+    def list_all_items(self, list_id: str, page: int = None, page_size: int = None):
+        params = self._check_pagination_params(page, page_size)
+        return self._client.get(
+            self._client.proactive_connect_host(),
+            f'/v0.1/bulk/lists/{list_id}/items',
+            params,
+            auth_type=self._auth_type,
+        )
+
+    def create_item(self, list_id: str, data: dict):
+        params = {'data': data}
+        return self._client.post(
+            self._client.proactive_connect_host(),
+            f'/v0.1/bulk/lists/{list_id}/items',
+            params,
+            auth_type=self._auth_type,
+        )
+
+    def get_item(self, list_id: str, item_id: str):
+        return self._client.get(
+            self._client.proactive_connect_host(),
+            f'/v0.1/bulk/lists/{list_id}/items/{item_id}',
+            auth_type=self._auth_type,
+        )
+
+    def update_item(self, list_id: str, item_id: str, data: dict):
+        params = {'data': data}
+        return self._client.put(
+            self._client.proactive_connect_host(),
+            f'/v0.1/bulk/lists/{list_id}/items/{item_id}',
+            params,
+            auth_type=self._auth_type,
+        )
+
+    def delete_item(self, list_id: str, item_id: str):
+        return self._client.delete(
+            self._client.proactive_connect_host(),
+            f'/v0.1/bulk/lists/{list_id}/items/{item_id}',
+            auth_type=self._auth_type,
+        )
+
+    def download_list_items(self, list_id: str, file_path) -> List[dict]:
+        uri = f'https://{self._client.proactive_connect_host()}/v0.1/bulk/lists/{list_id}/items/download'
+        logger.debug(f"GET request sent to {repr(uri)}")
+        headers = {**self._client.headers, 'Authorization': self._client._create_jwt_auth_string()}
+        response = requests.get(
+            uri,
+            headers=headers,
+        )
+        if 200 <= response.status_code < 300:
+            with open(file_path, 'wb') as file:
+                file.write(response.content)
+        else:
+            return self._client.parse(self._client.proactive_connect_host(), response)
+
+    def upload_list_items(self, list_id: str, file_path: str):
+        uri = f'https://{self._client.proactive_connect_host()}/v0.1/bulk/lists/{list_id}/items/import'
+        with open(file_path, 'rb') as csv_file:
+            logger.debug(f"POST request sent to {repr(uri)}")
```

Comment:
Not a blocker, I would make this a bit more distinguished from the client message. Something like `POST proactive connect uploading {file_path} to {repr(uri)}`

Reply:
Good call, I've made that change now.